File: security/src/main/java/ru/virgil/security/firebase/FirebaseSecurityProperties.java
package ru.virgil.security.firebase;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "security.firebase")
public record FirebaseSecurityProperties(
        String adminCredentialsFilePath,
        String webCredentialsFilePath,
        String authPageTemplateName
) {

}

File: security/src/main/java/ru/virgil/security/firebase/FirebaseSecurityProperties.kt
package ru.virgil.security.firebase

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "security.firebase")
data class FirebaseSecurityProperties(
    var adminCredentialsFilePath: String?,
    var webCredentialsFilePath: String?,
    var authPageTemplateName: String?,
)
